downstream/assemblies/platform/assembly-gw-resources.adoc

```diff
@@ -0,0 +1,18 @@
+ifdef::context[:parent-context: {context}]
+
+:_mod-docs-content-type: ASSEMBLY
+
+[id="assembly-gw-resources_{context}"]
+
+= Resources
+
+:context: access-mgmt-resources
+
+You can manage who has access to {PlatformNameShort} resources and what users can do with those resources. Users are granted access through the roles to which they are assigned or through roles inherited through the role hierarchy, for example, through the roles they inherit through team membership. {PlatformNameShort} resources differ depending on the functionality you are configuring. For example, resources can be job templates and projects for automation execution or decision environments and rulebook activations for automation decisions.
```

Comment:
I'd change "who has access" to "user access"

Reply:
Fixed.